### DISCUSSION TITLE
increase the size/scale of the UI at runtime

### DISCUSSION THREAD

**Discussion by alexzanderr** [DioxusLabs/dioxus#713] [Q&A]
hello

you know that on a webpage in browser you can `ctrl+=` and `ctrl+-` to increase and decrease the scale/size of the UI.

`ctrl+=`, `ctrl+-`, `ctrl+shift+=` and `ctrl+shift+-` dont seem to resize the contents inside the UI.

is it possible? how can i do that?

thanks

**Comment by aimerib** [comment]
This should be possible, but I haven't seen an example doing exactly that yet. However, there are two relevant examples that can help you here. The `all_events.rs` example can show you how to watch for keyboard events, and you can listen for key combinations like this (from that example):

`KeyPress(UiEvent { bubble_state: Cell { value: true }, data: KeyboardData { key: Character("-"), code: Minus, modifiers: CONTROL, location: Standard, is_auto_repeating: false } })`

or the ctrl+shift+- and ctrl+shift++ :
`KeyUp(UiEvent { bubble_state: Cell { value: true }, data: KeyboardData { key: Character("_"), code: NumpadAdd, modifiers: CONTROL | SHIFT, location: Numpad, is_auto_repeating: false } })

`KeyUp(UiEvent { bubble_state: Cell { value: true }, data: KeyboardData { key: Character("+"), code: NumpadAdd, modifiers: CONTROL | SHIFT, location: Numpad, is_auto_repeating: false } })`

and combine that with the `window_zoom.rs` example where zoom is being applied to the window in the same way as the browser (zoom is global, as opposed to a contained area).

This might be a little bit too low level for what you were looking for, but the lego pieces are there, they just need to be pieced together.

**Reply by alexzanderr** [reply]
nice! thanks for the response! i will try this out.

**Reply by alexzanderr** [reply]
yes. its working. thanks a lot!

but i need `dioxus_desktop 0.3.0` to support `webview`.

is it possible to get the `webview` field using `dioxus_desktop 0.2.3` ?
